tests/sentry/sentry_metrics/querying/data_v2/test_api.py

```diff
@@ -263,7 +263,7 @@ def test_query_with_group_by(self) -> None:
         first_meta = sorted(meta[0], key=lambda value: value.get("name", ""))
         assert first_meta[0] == {
             "group_bys": ["platform", "transaction"],
-            "limit": None,
+            "limit": 10000,
```

Comment:
This was fixed, since in the previous implementation we were not setting correctly the limit.